json_schema_to_class.py

```diff
@@ -154,15 +154,19 @@ def is_inner_model(self):
         return not isinstance(self.items, Basic)
 
     def to_init_code(self) -> str:
+        Array.use_list = True
         if not self.is_inner_model():
-            Array.use_list = True
             return '{spaces}self.{name}: {type_name} = values.get("{name}", {default})'.format(
                 spaces=spaces(2),
                 name=self.name,
                 type_name=f'List[{self.items.type_name()}]',
                 default=repr(self.default)
             )
-
+        if isinstance(self.items, Definition):
+            return (
+                f'{spaces(2)}self.{self.name}: List[{self.items.class_name()}] = '
+                f'self.{self.class_name()}(values=values.get("{self.name}"))'
+            )
         return f'{spaces(2)}self.{self.name} = self.{self.class_name()}(values=values.get("{self.name}"))'
```

Comment:
It looks like now this line is never called the (So does CodeCov says), but I am not sure if I can remove that. Please let me know, or explain to me how to add a test that reach here.